tests/integration/conftest.py

```diff
@@ -93,22 +110,77 @@ async def app(
     # mypy doesn't see that ActiveStatus has a name
     await model.wait_for_idle(status=ActiveStatus.name)  # type: ignore
 
+    # test that the application is online
```

Comment:
Can these values be made configurable or it doesn't make sense? I mean try 10 times to call the app and wait for 10 seconds between tries.

Reply:
I'm not sure it makes sense. It's arbitrary. I just wanted to give the mattermost instance a bit of time to come online and not wait for too long.

Reply:
I used David's proposition